src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -302,6 +344,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         {
             StorageApplicationPermissions::FutureAccessList().AddOrReplace(L"PickedFolderToken", folder);
             _State.Profile().StartingDirectory(folder.Path());
+            StartingDirectoryUseParentCheckbox().IsChecked(false);
```

Comment:
Changing the starting directory should automatically trigger an update to the checkbox